tests/test_nsjail.py

```diff
@@ -280,3 +282,38 @@ def test_py_args(self):
 
         self.assertEqual(result.returncode, 0)
         self.assertEqual(result.args[-3:-1], args)
+
+
+class NsJailArgsTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls._temp_dir = tempfile.TemporaryDirectory()
+        cls.addClassCleanup(cls._temp_dir.cleanup)
+
+        cls.nsjail_path = shutil.copy2("/usr/sbin/nsjail", cls._temp_dir.name)
+        cls.config_path = shutil.copy2("./config/snekbox.cfg", cls._temp_dir.name)
+        cls.max_output_size = 1_234_567
+        cls.read_chunk_size = 12_345
+
+        cls.nsjail = NsJail(
+            cls.nsjail_path, cls.config_path, cls.max_output_size, cls.read_chunk_size
+        )
```

Comment:
Should this really be shared across tests? I think this should be placed in `def setUp(self):` instead.

Reply:
The instance is designed to be re-used but I suppose it is safer to isolate test dependencies. 308385c